api/src/config.js

```diff
@@ -17,6 +17,7 @@ const configSchema = Joi.object({
       smtp: Joi.object({
         host: Joi.string().required(),
         port: Joi.number().default(587),
+        starttls: Joi.boolean().default(true),
```

Comment:
Please adjust the defaults here as well.

Reply:
Done as requested